slsim/Plots/lens_plots.py

```diff
@@ -80,6 +81,9 @@ def rgb_image(self, lens_class, rgb_band_list, add_noise=True):
             minimum = 0
             stretch = 0.5
             Q = 8
+        minimum = kwargs.get("minimum", minimum)
```

Comment:
I don't like extracting explicitly arguments out of a dictionary. It is unclear for the user which ones can/need to be set and/or what the default values are.
I would put minimum, stretch and Q as inputs with =None, and if None, set them according to the previous definition